### ISSUE TITLE
Upgrade to 2.9.2， java.lang.NumberFormatException: For input string: ""

### ISSUE THREAD

**Issue by jameycai** [springfox/springfox#2776]
Upgrading springfox-swagger2  to 2.9.2, accessing the following error：

2018-11-12 11:21:44.564 WARN  i.s.m.p.AbstractSerializableParameter - Illegal DefaultValue 50 for parameter type integer
java.lang.NumberFormatException: For input string: ""
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Long.parseLong(Long.java:601)
	at java.lang.Long.valueOf(Long.java:803)
	at io.swagger.models.parameters.AbstractSerializableParameter.getExample(AbstractSerializableParameter.java:412)
	at sun.reflect.GeneratedMethodAccessor100.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:687)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:719)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:155)
	at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serializeContents(IndexedListSerializer.java:119)
	at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:79)
	at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:18)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:727)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:719)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:155)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:727)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:719)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:155)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serializeFields(MapSerializer.java:718)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:639)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:33)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:727)

**Comment by DreamXiao** [comment]
https://github.com/springfox/springfox/issues/2265#issuecomment-413286451
#2265

**Comment by DreamXiao** [comment]
Duplicate of #2265

**Comment by DreamXiao** [comment]
2.9.2版本好像是这样。

**Comment by jiansong0720** [comment]
2.9.2 has this bug.i do not find a way to solve it.if you has solved it,tell me.thanks

**Comment by chrisinmtown** [comment]
Just copying from #2265: springfox version 2.9.2 pulls in swagger-models and swagger-annotations libraries at version 1.5.20, you can silence this annoying warning in your log by overriding to pull versions 1.15.21 instead.

**Comment by blankhang** [comment]
in application.yml
```java
logging:
    io:
      swagger:
        models:
          parameters:
            AbstractSerializableParameter: error
```

**Comment by PRASANTHRAJENDRAN** [comment]
In Gradle, we can solve this issue as below

		```
//springfox-swagger dependencies
		implementation('io.springfox:springfox-swagger2:2.9.2') {
			exclude group: 'io.swagger', module: 'swagger-annotations'
			exclude group: 'io.swagger', module: 'swagger-models'
		}
		implementation 'io.springfox:springfox-swagger-ui:2.9.2'
		implementation 'io.swagger:swagger-annotations:1.5.21'
		implementation 'io.swagger:swagger-models:1.5.21'
		implementation 'io.springfox:springfox-bean-validators:2.9.2'
```

**Comment by dilipkrish** [comment]
This is fixed as part of the 3.0.0 release
